### ISSUE TITLE
Create a prefilled issue link in the logs with the relevant session & debug data ready to go.

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1896]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.9.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.9.0...4.9.1) (2021-07-13)

- 🥚  patches update. fixes #1897 [`#1897`](https://github.com/open-wa/wa-automate-nodejs/issues/1897)
- 📝  resolve documentation 404'ing #1891 [`3b2f586`](https://github.com/open-wa/wa-automate-nodejs/commit/3b2f586fc6f463b61a83b87e6455f4523b678b98)
- Release 4.9.1 [`752c5c4`](https://github.com/open-wa/wa-automate-nodejs/commit/752c5c4940fe74375d44a92a3789ead0b05d46e0)
- create bug_report.yaml #1895 [`f43e4af`](https://github.com/open-wa/wa-automate-nodejs/commit/f43e4af648ced4628b1badbfa5e95afa5362c4de)
- 🔧 delete bug_report.md #1895 [`06d7f12`](https://github.com/open-wa/wa-automate-nodejs/commit/06d7f12962c18e04a429b8bc67a43f4c7d4f1f41)
- 🔒️ implemented mechanism to prevent CDN ddosing from deprecated versions #1893 [`85d629c`](https://github.com/open-wa/wa-automate-nodejs/commit/85d629c5e0dcbaa05fa73840c86c66d687d1fb5f)
- 🔧 fix bug_report.yaml #1895 [`f2db09c`](https://github.com/open-wa/wa-automate-nodejs/commit/f2db09c32b0372d6350491861460bca38ecb76d1)
- ⚡️ Only attempt to preload patches on VALID sessions #1893 [`c1100d9`](https://github.com/open-wa/wa-automate-nodejs/commit/c1100d919b253e723533391435d722d3d6d26e56)
- ✨ logs: added easy pre-filled issue template link to launch logs #1896 [`a0777e6`](https://github.com/open-wa/wa-automate-nodejs/commit/a0777e6a3be43d8d5a205b8bb7d9b5bd0e084f5b)
- 🥚  patches update [`112b862`](https://github.com/open-wa/wa-automate-nodejs/commit/112b8623f4b18d96825770380ed1fc2e35b9bd04)
- 🥚  patches update [`9094be5`](https://github.com/open-wa/wa-automate-nodejs/commit/9094be5acb23a7993dd330bb059b48e320ccae0e)
- 🥚  patches update [`d7f7ea0`](https://github.com/open-wa/wa-automate-nodejs/commit/d7f7ea010b5b4e8b21d9b02fb23029115628ba6b)
- 🥚  patches update [`978a71e`](https://github.com/open-wa/wa-automate-nodejs/commit/978a71e1af7e1c6ae0f2ac2fd0a5d1752350f055)
- Update bug_report.yaml [`623b438`](https://github.com/open-wa/wa-automate-nodejs/commit/623b4383476f9359c24783a68c6b46abfca61188)
